src/services/lease-service/routes/contacts.ts

```diff
@@ -171,6 +172,55 @@ export const routes = (router: KoaRouter) => {
     }
   })
 
+  /**
+   * @swagger
+   * /tenant/contactCode/{contactCode}:
+   *   get:
+   *     summary: Gets tenant by contact code
+   *     description: Retrieve tenant information by contact code.
+   *     tags: [Tenants]
+   *     parameters:
+   *       - in: path
+   *         name: contactCode
+   *         required: true
+   *         schema:
+   *           type: string
+   *         description: The contact code of the tenant.
+   *     responses:
+   *       200:
+   *         description: Successfully retrieved tenant information.
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 data:
+   *                   type: object
+   *                   description: The tenant data.
+   *       404:
+   *         description: Not found.
+   *       500:
+   *         description: Internal server error. Failed to retrieve Tenant information.
+   */
+  router.get('(.*)/tenant/contactCode/:contactCode', async (ctx) => {
```

Comment:
`tenants` to align with DIGG-standard

Reply:
Is that to add the `metadata` property to the response?

Reply:
https://github.com/Bostads-AB-Mimer/onecore-leasing/pull/109/commits/bbb24996e46cc910afa24c357bd70e477f808ae1